### ISSUE TITLE
winget upgrade for nvm is broken

### ISSUE THREAD

**Issue by remidebette** [coreybutler/nvm-windows#876]
**Issue:**
Hi, thank you for the nice tool.

I am installing NVM for windows with winget, which is becoming the de facto tool for managing packages in Windows.

```powershell
❯ winget install --id CoreyButler.NVMforWindows
```

My issue is that to this date I cannot seamlessly use the upgrade function of winget due to NVM for windows not been upgraded by the tool:

```powershell
❯ winget upgrade --id CoreyButler.NVMforWindows
This package's version number cannot be determined. To upgrade it anyway, add the argument --include-unknown to your previous command.
```
Adding the `--include-unknown` parameter is not a good solution because I am never not in a state where the system is 'up to date'

You can see the main reason is that winget is unable to detect the current version of NVM for windows:
```powershell
❯ winget upgrade --include-unknown
Name                                 Id                        Version Available Source
---------------------------------------------------------------------------------------
NVM for Windows                      CoreyButler.NVMforWindows Unknown 1.1.9     winget
```

**Expected Behavior:**
The actual version of the currently installed version should appear as so:
```powershell
❯ winget upgrade --include-unknown
Name                                 Id                        Version Available Source
---------------------------------------------------------------------------------------
NVM for Windows                      CoreyButler.NVMforWindows 1.1.9   1.1.9     winget
```

**Additional context:**
As explained here: https://github.com/microsoft/winget-cli/issues/1356
The version used by winget is simply the information provided from Control Panel
Apparently, all that is needed is to add a DisplayVersion key in the registry when installing.

See how the Control panel does not see the version:
![image](https://user-images.githubusercontent.com/16523229/199259475-5255044f-5eef-4d33-9dc5-788151d8a287.png)

Here is an example of a program with a version:
![image](https://user-images.githubusercontent.com/16523229/199259329-49293882-c439-48d5-a76a-6c71313d5ee2.png)

**Comment by alpturedi** [comment]
I think if you use the update utility it doesn't work so you probably need to use the installer to fix this.

**Comment by alpturedi** [comment]
> I think if you use the update utility it doesn't work so you probably need to use the installer to fix this.

I can confirm that if you use the installer to update the version it fixes the issue.

**Comment by remidebette** [comment]
This bug has been fixed in 1.1.10
When available from Winget, `winget upgrade --include-unknown` will also work
